components/compliance-service/integration_test/stats_server_read_trend_test.go

```diff
@@ -0,0 +1,187 @@
+package integration_test
+
+import (
+	"testing"
+
+	iam_v2 "github.com/chef/automate/api/interservice/authz/v2"
+	"github.com/chef/automate/components/compliance-service/ingest/events/compliance"
+
+	apiReporting "github.com/chef/automate/components/compliance-service/api/reporting"
+	reportingServer "github.com/chef/automate/components/compliance-service/api/reporting/server"
+	"github.com/chef/automate/components/compliance-service/api/stats"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	authzConstants "github.com/chef/automate/components/authz-service/constants/v2"
+	statsServer "github.com/chef/automate/components/compliance-service/api/stats/server"
+	"github.com/chef/automate/components/compliance-service/reporting/relaxting"
+)
+
+func TestReadTrend(t *testing.T) {
+	reportFileName := "../ingest/examples/compliance-success-tiny-report.json"
+	everythingCtx := contextWithProjects([]string{authzConstants.AllProjectsExternalID})
+
+	statsServer := statsServer.New(&relaxting.ES2Backend{ESUrl: elasticsearchUrl})
+	reportingServer := reportingServer.New(&relaxting.ES2Backend{ESUrl: elasticsearchUrl})
+
+	n := 5
+
+	reportIds := make([]string, n)
+
+	for i := 0; i < n; i++ {
+		err := suite.ingestReport(reportFileName, func(r *compliance.Report) {
+			id := newUUID()
+
+			r.Environment = id
+			r.NodeName = id
+			r.NodeUuid = id
+			r.Platform.Name = id
+			r.Profiles[0].Controls = r.Profiles[0].Controls[:1]
+			r.Profiles[0].Controls[0].Id = id
+			r.Profiles[0].Controls[0].Title = id
+			r.Profiles = r.Profiles[:1]
+			r.Profiles[0].Sha256 = id
+			r.Profiles[0].Title = id
+			r.Recipes = []string{id}
+			r.ReportUuid = id
+			r.Roles = []string{id}
+
+			reportIds[i] = id
+		})
+
+		require.NoError(t, err)
+	}
+
+	defer suite.DeleteAllDocuments()
+
+	waitFor(func() bool {
+		response, _ := reportingServer.ListReports(everythingCtx, &apiReporting.Query{})
+
+		return response != nil && len(response.Reports) == n
+	})
+
+	reportsProjects := map[string][]string{
+		"project1": reportIds[1:3],
+		"project2": reportIds[2:5],
+		"project3": reportIds[3:],
+	}
+
+	projectRules := map[string]*iam_v2.ProjectRules{}
+	for k, v := range reportsProjects {
+		projectRules[k] = &iam_v2.ProjectRules{
+			Rules: []*iam_v2.ProjectRule{
+				{
```

Comment:
You should add the below line even though your test is bypassing the check for it. 
```suggestion
				{
					Type: iam_v2.ProjectRuleTypes_EVENT,
```